strings/en_US/messages.po

```diff
@@ -3588,3 +3582,342 @@ msgstr "Manage Users"
 
 msgid "Manage Events"
 msgstr "Manage Events"
+
+msgid "Please answer the following questions correctly so that our community team can verify all new Champions are real people."
+msgstr "Please answer the following questions correctly so that our community team can verify all new Champions are real people."
+
+msgid "Congratulations on registering as a CoderDojo Champion!"
+msgstr "Congratulations on registering as a CoderDojo Champion!"
+
+msgid "A Champion is an individual who volunteers to take charge of setting up, running and maintaining a Dojo."
+msgstr "A Champion is an individual who volunteers to take charge of setting up, running and maintaining a Dojo."
+
+msgid "As a Champion you care about your community enough to take the initiative to start a Dojo and give youth the opportunity to learn to code and be creative with technology!"
+msgstr "As a Champion you care about your community enough to take the initiative to start a Dojo and give youth the opportunity to learn to code and be creative with technology!"
+
+msgid "First name"
+msgstr "First name"
+
+msgid "Last name"
+msgid "Last name"
+
+msgid "Personal/Work email address"
+msgstr "Personal/Work email address"
+
+msgid "Date of birth"
+msgstr "Date of birth"
+
+msgid "Parent Email address"
+msgstr "Parent Email address"
+
+msgid "Parent Name"
+msgstr "Parent Name"
+
+msgid "Home address"
+msgstr "Home address"
+
+msgid "Phone number"
+msgstr "Phone number"
+
+msgid "Telephone number is not valid"
+msgstr "Telephone number is not valid"
+
+msgid "(optional)"
```

Comment:
Well, it's already an issue with ltr because our display separate this string from its context..